tests/api.c

```diff
@@ -33597,8 +33597,240 @@ static int test_wc_RNG_GenerateBlock(void)
 }
 #endif
 /*
- * Testing wc_InitRngNonce
+ * Testing get_rand_digit
  */
+static int test_get_rand_digit (void)
+{
+    int ret = 0;
+#if !defined(WC_NO_RNG) && defined(WOLFSSL_PUBLIC_MP)
+
+    WC_RNG      rng;
+    mp_digit    d;
+
+    printf(testingFmt, "get_rand_digit()");
+
+    ret = wc_InitRng(&rng);
+
+    if (ret == 0) {
+        ret = get_rand_digit(&rng, &d);
```

Comment:
In addition to this expected success case, let's try some bad argument checks as well to make sure things fail as expected.